roles/container-engine/cri-o/tasks/reset.yml

```diff
@@ -17,7 +17,7 @@
 - name: CRI-O | Remove cri-o apt repo
   apt_repository:
     repo: "deb {{ crio_download_crio }}{{ crio_version }}/{{ crio_kubic_debian_repo_name }}/ /"
-    state: present
+    state: absent
```

Comment:
After the commit 5ebd305d17d1b62d20f695448f8fc0cae45fc465 this part only seems necessary to be merged.

Reply:
merged master into this PR to solve your review